t/ui/27-plugin_obs_rsync_selenium.t

```diff
@@ -0,0 +1,152 @@
+# Copyright (C) 2019 SUSE LLC
```

Comment:
I suggest to keep "selenium" out of the filename. All our test files in `t/ui/` can be expected to use "Selenium" so this as part of the filename is redundant.

Reply:
The only difference of this case from the other obs_rsync* tests is that it uses selenium, so current file name perfectly describes what the test does.

Reply:
I see. I guess the other tests shouldn't even be in "ui" then.

Reply:
Why? Most of test in t/ui/ don't use Selenium

Reply:
Correction: not `most` but `12 out of 33`, but still significant part. You don't require Selenium to test UI and I think the folder must be named `selenium` if you expect only selenium tests inside it